cafebabel/templates/base.html

```diff
@@ -7,6 +7,7 @@
 
 {% block metas_social %}{% endblock %}
 
+{% block extra_head %}{% endblock %}
```

Comment:
…to re-swap this block below the `style.css` inclusion.

Reply:
ok ! got it !